lib/kubeclient/common.rb

```diff
@@ -265,7 +265,7 @@ def get_entities(entity_type, klass, resource_name, options = {})
       resource_version = result.fetch('resourceVersion', nil)
       if resource_version.nil?
         resource_version =
-            result.fetch('metadata', {}).fetch('resourceVersion', nil)
+          result.fetch('metadata', {}).fetch('resourceVersion', nil)
```

Comment:
@grosser fixing the indentation style (here and below) is not related to this PR. Please send a separate PR (which I suppose will be merged much faster).